CIでStorybookのテストが確率的に失敗する

失敗している例：

```
> Run bun run storybook:test:ci
$ concurrently --kill-others --success=first --names='SERVER,TEST' --prefix-colors='gray,blue' 'bun storybook --quiet --no-open' 'wait-on tcp:6006 && bun storybook:test'
[SERVER] $ storybook dev -p 6006 --quiet --no-open
[SERVER] @storybook/cli v[8](https://github.com/r4ai/yumemi-passport-frontend/actions/runs/8982522534/job/24670313402#step:6:9).0.10
[SERVER] 
[SERVER] (node:5131) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
[SERVER] (Use `node --trace-deprecation ...` to show where the warning was created)
[TEST] $ test-storybook --coverage
[TEST] (node:5155) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
[TEST] (Use `node --trace-deprecation ...` to show where the warning was created)
[TEST] [test-storybook] It seems that your Storybook instance is not running at: http://127.0.0.1:6006. Are you sure it's running?
[TEST] 
[TEST] If you're not running Storybook on the default 6006 port or want to run the tests against any custom URL, you can pass the --url flag like so:
[TEST] 
[TEST] yarn test-storybook --url http://127.0.0.1:[9](https://github.com/r4ai/yumemi-passport-frontend/actions/runs/8982522534/job/24670313402#step:6:10)009
[TEST] 
[TEST] More info at https://github.com/storybookjs/test-runner#getting-started
[TEST] error: script "storybook:test" exited with code 1
[TEST] wait-on tcp:6006 && bun storybook:test exited with code 1
--> Sending SIGTERM to other processes..
[SERVER] bun storybook --quiet --no-open exited with code SIGTERM
error: script "storybook:test:ci" exited with code 1
Error: Process completed with exit code 1.
```

https://github.com/r4ai/yumemi-passport-frontend/actions/runs/8982522534/job/24670313402

原因は恐らく、Storybookのサーバーが完璧に立ち上がる前にテストを実行しているためだと思われる。`wait-on`コマンドにより`tcp:6006`が利用可能になるまで待つようにしているが、`tcp:6006`が利用可能になってからもサーバーが完璧に立ち上がるまでに少しのラグがある可能性がある。そこで、`wait-on`の`--delay`オプションで適当な遅延を入れると良いと思われる。

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "storybook": "storybook dev -p 6006",
     "storybook:build": "storybook build",
     "storybook:test": "test-storybook --coverage",
-    "storybook:test:ci": "concurrently --kill-others --success=first --names='SERVER,TEST' --prefix-colors='gray,blue' 'bun storybook --quiet --no-open' 'wait-on tcp:6006 && bun storybook:test'",
+    "storybook:test:ci": "concurrently --kill-others --success=first --names='SERVER,TEST' --prefix-colors='gray,blue' 'bun storybook --quiet --no-open' 'wait-on --delay=500 tcp:6006 && bun storybook:test'",
     "prepare": "lefthook install && panda codegen"
   },
   "dependencies": {

